staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -176,6 +177,16 @@ var (
 			Help:           "Counter for consistent reads from cache.",
 			StabilityLevel: compbasemetrics.ALPHA,
 		}, []string{"resource", "success", "fallback"})
+
+	storageDigestDescription = compbasemetrics.NewDesc(
+		"apiserver_storage_digest",
+		"Gage exposing digest of storages to validate consistency between. Algorithm to calculate digest might change, so it should not be compared across apiservers.",
```

Comment:
Hmm - I'm wondering what's the usecase for the metric - how do we envision to be used?

Reply:
In an alert you won't be able to  compare the digests because you can't compare label values with promQL.

How about exposing something like that instead:
```
apiserver_storage_digests_mismatches_total
```
where you directly expose the number of mismatches?

Then when there is a mismatch, you could log the digests if you think the information would be relevant to further investigate the issue.

Reply:
Updated, please take a look.